test/unified_format.py

```diff
@@ -142,28 +149,52 @@ def parse_bulk_write_error_result(error):
     return parse_bulk_write_result(write_result)
 
 
-class EventListenerUtil(CommandListener):
+class NonLazyCursor(object):
```

Comment:
NonLazyCursor is used for the new createFindCursor operation.

Reply:
I am not sure I follow - why is this needed?

Reply:
createFindCursor mandates that the cursor is actually created on the server but not iterated: https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#createfindcursor